tests/e2e/default-mobile/contacts/barcode-search-contacts.wdio-spec.js

```diff
@@ -0,0 +1,50 @@
+const searchPage = require('@page-objects/default-mobile/search/search.wdio.page.js');
```

Comment:
Hi @lorerod just popping up here to see if you need my help with the barcode?

Reply:
Hi, @latin-panda. We are experiencing a strange behavior when [running the e2e test](https://github.com/medic/cht-core/pull/8732#issuecomment-1843357767) compared to when[ running the developer setup first and then the e2e test](https://github.com/medic/cht-core/pull/8732#issuecomment-1843642985). When running the e2e test directly in the branch, it does not use the latest changes, and the test fails. This is also happening in CI.

Reply:
The version in `package.json` and `package-lock.json` is different from the [4.4.1-FR-barcode](https://github.com/medic/cht-core/blob/a17e8987a93aee1d042a57e5fa917d79f42e4769/package.json#L3) branch. We need to pull the latest from `4.4.1-FR-barcode`

Reply:
Thanks @latin-panda! I pulled the latest from `4.4.1-FR-barcode`, but the test still [fails](https://github.com/medic/cht-core/actions/runs/7389173632?pr=8732) on CI because the barcode search icon doesn't show. I will continue looking into this to see if I can find the problem.